guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -1,55 +1,267 @@
 [id="host-substatus-overview_{context}"]
 = Host sub-status overview
 
-A sub-status monitors only part of a host's capabilities.
+A sub-status monitors only a part of a host's capabilities.
 
-Currently, {Project} ships only with *Build* and *Configuration* sub-statuses.
-There can be more sub-statuses depending on which plug-ins you add to your {Project}.
+To view the sub-statuses of a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and click on the host whose full status you want to inspect.
+You can view the global host status next to the name of the host and the host sub-statuses on the *Host status* card.
 
-The *build* sub-status is relevant for managed hosts.
+Each sub-status can define its own set of possible values that are mapped to the three global status values.
 
-The *configuration* sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
+Below are listed sub-statuses that {Project} contains.
+ifdef::foreman-el,foreman-deb,katello[]
+There can be more sub-statuses depending on which plug-ins you add to your {Project}.
+endif::[]
 
-To view the sub-status for a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and click the host whose full status you want to inspect.
-You can also view substatus information in the hover help for each host.
+Configuration::
+This sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status
+| Alerts disabled | OK
+| Active | OK
+| Pending | OK
+| No changes | OK
+| No reports | OK / Warning
+| Out of sync | Warning
+| Error | Error
+|===
++
+Additional information about the values of this sub-status:
++
+* *Active*: During the last configuration, some resources were applied.
+* *Pending*: During the last configuration, some resources would be applied but your configuration management integration was configured to run in `noop` mode.
+* *No changes*: During the last configuration, nothing changed.
+* *No reports*: This can be both a *Warning* or *OK* status.
+When there are no reports but the host uses an associated configuration management proxy or the `always_show_configuration_status` setting is set to `true`, it maps to *Warning*.
```

Comment:
what do you mean by "proxy"? Maybe reword to "{SmartProxy} for configuration management"?

Reply:
Reworded.